polygon-digital-carbon/schema.graphql

```diff
@@ -376,6 +385,24 @@ type ToucanBatch @entity {
   creationTransactionHash: Bytes!
 }
 
+type C3OffsetRequest @entity {
```

Comment:
Should we not call this a `Retire` request? Or does C3's own language in the contracts call it "offset" as the verb?

Reply:
in the C3 contracts the function is `offsetFor` which is why I initially named it this way. Changed to Retire though to stay consistent on the Carbonmark side.